packages/block-editor/src/text-input/sidebar.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import {
+	// eslint-disable-next-line
+	__experimentalToggleGroupControl as ToggleGroupControl,
+	// eslint-disable-next-line
+	__experimentalToggleGroupControlOption as ToggleGroupControlOption,
```

Comment:
Those two confused me a bit. Checking the `@wordpress/components` storybook, I could see that `ToggleGroupControl` and `ToggleGroupControlOption` are not under the experimental section. Should we update our `@wordpress/components` version?

Reply:
Actually, I added those two. Although they are not under the experimental section on Storybook, even in the latest version, the packages are exported as experimental:

https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/index.js#L138

Reply:
Let's keep it then 👍